judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ItemEssayCard/ItemEssayForm/ItemEssayForm.scss

```diff
@@ -0,0 +1,34 @@
+.item-essay-form {
```

Comment:
BTW, just realized. Please put all other css classes inside the main css class (`.item-essay-form`), otherwise all components that use class with the same name will also pick up the class.

Reply:
(and in all your previous PRs, please double check)

Reply:
Already fixed in next commit